README.md

```diff
@@ -31,24 +33,71 @@ Besides [Rust](https://www.rust-lang.org/learn/get-started#installing-rust) itse
 - `pkg-config`
 - `libssl-dev` (in Debian/Ubuntu) or `openssl-dev` (in Fedora/Red Hat)
 
+### From crates.io
+
+To download from [crates.io](https://crates.io), compile and install the client:
+
+```bash
+cargo install nimiq-client
```

Comment:
Does that use nightly automatically?

Reply:
That was actually still on my TODO list. I think you have to explicitly override it. I will fix this later.